docs/docs/streamr-network/incentives/network-incentives.md

```diff
@@ -30,6 +30,109 @@ The introduction of token economics defines three new roles: [Sponsor](../networ
+
+![Delegated tokens generate profit that is shared between protocol, delegators, and operator](@site/static/img/DATA-flows-Operator-contract.jpg)
+
+What's missing from the picture is the case where the profit sharing was due to a `triggerAnotherOperatorWithdraw` method call into a "fisherman's" Operator contract. The fisherman would receive a reward for finding another Operator that has too much unwithdrawn earnings. The reward is [`fishermanRewardFraction`](https://polygonscan.com/address/0x869e88dB146ECAF20dDf199a12684cD80c263c8f#readProxyContract) (currently set to 25%) of the withdrawn earnings. It's slashed from the operator's self-delegation and transferred to the fisherman's Operator contract.
+
+So the split of **2000 DATA** in case of fisherman-initiated withdraw would be:
+- protocol fee: **100 DATA**
+- operator's cut: **199 DATA**
+- profit: **1701 DATA**
+- fisherman's reward: 25% of earnings, or **500 DATA**
+
+And finally the operator's income would be 1049.5 DATA - 500 DATA = **549.5 DATA**. The fisherman's reward could even take the operator's income negative if the operator's cut is low enough.
+
+### Contract internal bookkeeping
+
+Sponsorship contracts hold different kinds of DATA that are internally kept separate:
+- `totalStakedWei`: total amount of tokens staked by all operators
+  - each operator has their `stakedWei`, part of which can be `lockedStakeWei` if there are flags on/by them
+  - stake belongs to the stakers, they can claim it via `unstake`, `forceUnstake` and `reduceStakeTo` methods
+  - the contract can slash the stake for early leaving or getting kicked after a vote
+- `remainingWei` that comes from the sponsor: part of the sponsorship that hasn't been paid out yet
+  - remaining sponsorship doesn't belong to anybody
+  - will be allocated to stakers according to the `StakeWeightedAllocationPolicy` when the contract is running
+- `earningsWei`: the part of the sponsorship that has been allocated to stakers but not yet withdrawn
+  - belongs to the stakers, they can claim it via `withdraw` method
+- `forfeitedStakeWei`: stakes that were locked to pay for a flag by a past operator who `forceUnstake`d (or was kicked)
+  - should be zero when there are no active flags
+
+Operator contract holds only "one kind" of DATA but itself functions as a "delegation token" [ERC-20 contract](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/). All DATA the Operator contract holds is considered "free funds" that the operator can use to stake into Sponsorship contracts. The only detail here is that the operator must first pay out the queued delegators (if any), though usually the payouts will happen automatically. Note that the Operator contract's DATA balance does not belong to the operator but to all of the delegators (including the operator via self-delegation) in proportion to their delegation.
```

Comment:
Is there a difference between _funds_ and _free funds_? It is so that when an operator stakes part of its funds, the free funds decrease?